weave/integrations/openai/openai_sdk.py

```diff
@@ -128,10 +131,10 @@ def _get_tool_calls(
 
 
 def openai_accumulator(
-    acc: Optional["ChatCompletionChunk"],
-    value: "ChatCompletionChunk",
+    acc: ChatCompletionChunk | None,
+    value: ChatCompletionChunk,
```

Comment:
this is imported in 

```
if TYPE_CHECKING:
    from openai.types.chat import ChatCompletionChunk
```

don't we need quotes still?

Reply:
Not if you do `from __future__ import annotations`

Reply:
https://peps.python.org/pep-0563/